Trakt/ScheduledTasks/SyncFromTraktTask.cs

```diff
@@ -171,7 +171,16 @@ private async Task SyncTraktDataForUser(Jellyfin.Data.Entities.User user, double
                     if (!userData.Played)
                     {
                         userData.Played = true;
-                        userData.LastPlayedDate = DateTimeOffset.UtcNow.UtcDateTime;
+
+                        if (!string.IsNullOrEmpty(matchedMovie.last_watched_at))
+                        {
+                            userData.LastPlayedDate = DateTimeOffset.Parse(matchedMovie.last_watched_at).ToUniversalTime().UtcDateTime;        
```

Comment:
Is there a specific reason you are using `DateTimeOffset.Parse` instead of `DateTime.TryParse` ?
I see it was set to `DateTimeOffset.UtcNow.UtcDateTime` before but I see no reason why not to use `DateTime` instead

Reply:
I copied it from [line 200](https://github.com/jellyfin/jellyfin-plugin-trakt/blob/d1f1f862dbd0fdbc0eee7a9dad8dff9a4d64e369/Trakt/ScheduledTasks/SyncFromTraktTask.cs#L200).

Blame says it was changed in [this commit](https://github.com/jellyfin/jellyfin-plugin-trakt/commit/de07336b2c99c0298dfb5dde0eb55f187bc2f2b0) but I don't know why.

Reply:
Trakt returns dates as UTC without timezones https://trakt.docs.apiary.io/#introduction/dates
`"last_watched_at": "2020-10-31T18:46:20.000Z"`
So I think DateTime should work as well.

Reply:
I can change it to that, should I?

Reply:
Yes that would be great

Reply:
Updated it, it still worked so I guess it's good.

Also tried to simplify some things.